config/settings/integration_tests.py

```diff
@@ -61,9 +61,15 @@
 # ------------------------------------------------------------------------------
 
 # article related
-ARTICLE_AUTHOR = env('ARTICLE_AUTHOR', "ArticleOwner")
-ARTICLE_AUTHOR_OVERRIDE = env('ARTICLE_AUTHOR_OVERRIDE', "ArticleContributorOverride")
-ARTICLE_BODY_CLASS = env('ARTICLE_BODY_CLASS', "sfdo-kb__body")
+ARTICLE_AUTHOR = env('ARTICLE_AUTHOR', default = "ArticleOwner")
+ARTICLE_AUTHOR_OVERRIDE = env('ARTICLE_AUTHOR_OVERRIDE', default = "ArticleContributorOverride")
+ARTICLE_BODY_CLASS = env('ARTICLE_BODY_CLASS', default = "sfdo-kb__body")
```

Comment:
Did you run black? Keyword arguments shouldn't have spaces around the equal sign, and the quotes are inconsistent.

Reply:
Black wanted to change basically every line of the file and I thought that keeping it "diff-friendly" with the other config file was a higher priority. I guess I could make a commit blackifying that whole directory later.

But I fixed the thing you noticed.

Reply:
Okay, doing a separate blackening sounds good.